examples/null.md

```diff
@@ -0,0 +1,141 @@
+# Null primitive type
+
+The `nullable` type attribute can be used to outline that `null` is an accepted value.
+
+## Using `null` with `string` and `array` types
+
+### MSON
+
+```mson
+- keyA (string, nullable, required) - Accepts `null` or `string`
```

Comment:
Again, I'd make sure it is obvious we are talking about JSON context in "Accepts `null` or `string`"

Maybe what about this:

```apib
- keyA (string, nullable, required) - Value of this key can be either unset or of the `string` type
```